trusted_application_demo/client_application/aes_certify_messaging/publisher/aes_publisher.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+* Copyright (c) 2018, ARM Limited.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "aes_custom_interface/srv/get_symm_secret.hpp"
+#include "aes_custom_interface/msg/aes_message_certify.hpp"
+
+#include <string>
+
+extern "C" {
+  #include <teec_utils.h>
+  #include <rsa_api.h>
+  #include <aes_api.h>
+  #include <hmac_api.h>
+  #include <tee_client_api.h>
+  #include <ta_public.h>
+}
+
+// Max number of byte for a message.
+#define PUBLISHER_MAX_MESSAGE_SIZE      18
+// Number of messages that will be sent by the publisher.
+#define PUBLISHER_MAX_NUMBER_MESSAGE   100
+
+using namespace std;
+using namespace placeholders;
+using namespace aes_custom_interface::srv;
+using namespace aes_custom_interface::msg;
```

Comment:
we recommend avoiding the usage of `using namespace` directives as per [google styleguide](https://google.github.io/styleguide/cppguide.html#Namespaces)